src/main/scala/EmrSparkPlugin.scala

```diff
@@ -17,10 +17,7 @@
 package sbtemrspark
 
 import com.amazonaws.services.elasticmapreduce.model.{Unit => _, _}
-import com.amazonaws.services.elasticmapreduce.{
-  AmazonElasticMapReduce,
-  AmazonElasticMapReduceClientBuilder
-}
+import com.amazonaws.services.elasticmapreduce.{AmazonElasticMapReduce, AmazonElasticMapReduceClientBuilder}
```

Comment:
There are many lines in this file that just changed the coding style without adding new features, along with many new added empty lines, did you apply your code formatter directly on the whole file? Would it be possible to revert these changes and only preserve the part that you really changed the logic?

Reply:
Done